tests/Database/Functional/Driver/Postgres/Query/InsertQueryTest.php

```diff
@@ -78,6 +78,19 @@ public function testSimpleInsertMultipleRows(): void
         );
     }
 
+    public function testInsertMultipleRowsAsArray(): void
+    {
+        $insert = $this->database->insert()->into('table')->values([
+            ['name' => 'Anton', 'balance' => 100],
+            ['name' => 'John', 'balance' => 200],
+        ]);
+
+        $this->assertSameQuery(
+            'INSERT INTO {table} ({name}, {balance}) VALUES (?, ?), (?, ?)',
+            $insert
+        );
+    }
```

Comment:
Looks like the same as `tests/Database/Functional/Driver/Common/Query/InsertQueryTest::testInsertMultipleRowsAsArray`

Reply:
Thanks, I deleted it.